src/PlaylistDetails/PlaylistDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Playlist } from '../utils';
+
+interface IProps { playlist: any }
+
+const displaySongs = (playlist: Playlist) => {
+  return playlist.tracks.reduce((finalSongs: JSX.Element[], currentSong, index: number): JSX.Element[] => {
+      const songToDisplay = 
+        <section key={index}>
+          <h3>{currentSong.songName}</h3>
+          <h3>{currentSong.artist.name}</h3>
+          <a target='_blank' href={currentSong.songUrl}>View song on Last FM</a>
+          <a target='_blank' href={currentSong.artist.artistUrl}>View Artist on Last FM</a>
+        </section>;
+      finalSongs.push(songToDisplay)
+    return finalSongs
+  }, [])
+}
```

Comment:
We may be able to expand more on it this the future. Currently, there isn't a ton of extra data on this page, but I think part of that is what the API data is currently returning. In order to make this work with more data, we would have to do another API call to Last.FM that would return more details on the specific album. Still have not been able to find an album cover picture so we will look more into this if we have the time.